pkgs/development/python-modules/manim-voiceover/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, pytest-xdist
+, pytestCheckHook
+, manim
+, mutagen
+, pydub
+, humanhash3
+, sox
+, pip
+, python-dotenv
+, gtts
+, pynput
+, pyaudio
+, playsound
+, stable-ts
+, pygobject3
+, gst_all_1
+, wrapGAppsHook
+, intltool
+}:
+
+buildPythonPackage rec {
+  pname = "manim-voiceover";
+  format = "pyproject";
+  version = "0.3.0";
+  disabled = pythonOlder "3.7";
+
+  extras = [ "gtts" "recorder" ];
+
+  src = fetchFromGitHub {
+    owner  = "ManimCommunity";
+    repo = "manim-voiceover";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-ugR435frhpleyzHG6cbKnTTyqRVpRKPBLkaZQ+E8Cwo=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    wrapGAppsHook
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'sox = "^1.4.1"' 'sox = "*"'
+  '';
+
+  propagatedBuildInputs = [
+    manim
+    mutagen
+    pydub
+    humanhash3
+    sox
+    python-dotenv
+    pip
+    gtts
+    playsound
+    pyaudio
+    pynput
+    stable-ts
+    pygobject3
+    intltool
+  ];
+
+  nativeCheckInputs = [
+    pytest-xdist
+    pytestCheckHook
+
+  ];
+
+  doCheck = false;
```

Comment:
Not sure what the test does https://github.com/ManimCommunity/manim-voiceover/blob/main/tests/test_install.py

Reply:
Tests just test the import like `pythonImportsCheck` so I am ignoring them.